lib/semian/circuit_breaker.rb

```diff
@@ -43,13 +49,16 @@ def mark_failed(error)
 
     def mark_success
       return unless half_open?
-      @successes += 1
+      #@successes += 1
```

Comment:
I had left it to keep the meaning clear, since writing a custom array container that only provides certain functionality (`:push`, `:pop`) while adding others that aren't normal array operations (`:successes`, etc) may leave the code confusing to read. If you think the meaning is still clear, then I'll remove it.